packages/charts/src/charts/metric/metric-chart.component.tsx

```diff
@@ -107,27 +111,19 @@ export const MetricChart: FC<Props> = ({
         <AnimatePresence>
           <motion.div {...textMotion}>
             <TextWrapper>
-              {valuePrefix && (
-                <Text data-test="metric-prefix" {...prefix.typography}>
-                  {valuePrefix}
-                </Text>
-              )}
-              <Text data-test="metric-value" {...valueSettings.typography}>
-                {formatValue ? formatValue(value) : value}
+              {valuePrefix && <Text {...prefix.typography}>{valuePrefix}</Text>}
+              <Text {...valueSettings.typography}>
+                {formatValue ? valueFormatter(value, formatValue) : value}
```

Comment:
The `valueFormatter` takes care about `undefined` formatValue - so this check for presence of `formatValue` is not required here.